src/minoltamn_int.cpp

```diff
@@ -1560,9 +1559,8 @@ static std::ostream& resolveLens0x80(std::ostream& os, const Value& value, const
     std::string model = getKeyString("Exif.Image.Model", metadata);
     std::string maxAperture = getKeyString("Exif.Photo.MaxApertureValue", metadata);
     long focalLength = getKeyLong("Exif.Photo.FocalLength", metadata);
-    std::string F4 = "1024/256";
 
-    if (model == "SLT-A77V" && maxAperture == F4 && inRange(focalLength, 18, 200)) {
+    if (model == "SLT-A77V" && maxAperture == "1024/256" && inRange(focalLength, 18, 200)) {
```

Comment:
Same here about losing contextual information. People is used to the F numbers

Reply:
Fixed.